app/javascript/packs/webauthn-setup.js

```diff
@@ -0,0 +1,57 @@
+function webauthn() {
+  const arrayBufferToBase64 = function(buffer) {
+    let binary = '';
+    const bytes = new Uint8Array(buffer);
+    const len = bytes.byteLength;
+    for (let i = 0; i < len; i += 1) {
+      binary += String.fromCharCode(bytes[i]);
+    }
+    return window.btoa(binary);
+  };
+  const longToByteArray = function(long) {
+    const byteArray = new Uint8Array(8);
+    for (let index = 0; index < byteArray.length; index += 1) {
+      const byte = long & 0xff;  // eslint-disable-line no-bitwise
+      byteArray[index] = byte;
+      long = (long - byte) / 256;
+    }
+    return byteArray;
+  };
+  const userId = document.getElementById('user_id').value;
+  const userEmail = document.getElementById('user_email').value;
```

Comment:
I wouldn't depend on the user email unless it's just for display purposes. We have tickets in the queue to move to multiple email addresses. If we do need to use an email address, then we should chat with the UX folk to figure out how we will select the email address. That's not a blocker for this PR, but something we need to be aware of going forward.